bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -18,4 +18,6 @@ Import-Package: groovy.lang,
  org.junit,
  org.junit.runner,
  org.junit.runners,
- org.osgi.service.cm
+ org.osgi.service.cm,
+ org.mockito
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
Require-Bundle is needed to resolve `org.hamcrest` correctly. Since it is a split package and none of the bundles providing parts of it (junit, mockito) do export the base package itself we have to hard reference the bundles which do here (this is at least my understanding and the only configuration I could get this to work). This is all required to have mockito available.